2-mandatory/7-eligible-students.js

```diff
@@ -14,7 +14,13 @@
   - Returns an array containing only the names of the who have attended AT LEAST 8 classes
  */
 
-function eligibleStudents(attendances) {}
+function eligibleStudents(attendance) {
+  return attendance
+    .filter((scores) => scores.attendance >= 8)
+    .map((theirNames) => theirNames.name);
+  // Anna I know you will read here 😂 I am getting it yes!😀 chaining methods.
```

Comment:
Yep you've mastered function chaining! Code is spot on, formatting/comment nice touch 🤣